src/rules/color-no-hex/__tests__/index.js

```diff
@@ -0,0 +1,69 @@
+import {
+  ruleTester,
+  warningFreeBasics,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule(undefined, tr => {
+  warningFreeBasics(tr)
+
+  tr.ok("a { color: pink; }")
+  tr.ok("a { color: rgba(0, 0, 0, 0); }")
+  tr.ok("a { color: #12345; }")
```

Comment:
I'd think that if I turned this rule on I would not want to also have to turn on `color-no-invalid-hex` --- so the *apparent attempts* at hex values should be flagged just as much as the valid hex values. What do you think about that?